src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -0,0 +1,982 @@
+# coding=utf-8
+# Copyright 2021 Facebook AI Research The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch MaskFormer model."""
+
+from __future__ import annotations
+
+import logging
+import math
+from dataclasses import dataclass, field
+from numbers import Number
+from typing import Any, Callable, Dict, List, Optional, Tuple, TypedDict, Union
+
+import numpy as np
+import torch
+import torchvision
+from torch import Tensor, nn
+from torch.nn import functional as F
+
+from einops import rearrange
+from einops.einops import repeat
+from timm.models.layers import DropPath, to_2tuple, trunc_normal_
+
+from ...file_utils import (
+    ModelOutput,
+    add_start_docstrings,
+    add_start_docstrings_to_model_forward,
+    is_scipy_available,
+    is_timm_available,
+    is_vision_available,
```

Comment:
Not sure any of those should be used in the modeling file.

Reply:
Removed